### PR TITLE
Updated 1.13 release notes

### PR DESCRIPTION
**Pull request by ajinrocket**
Signed-off-by: alexanderjin <ajin@rocketsoftware.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/getting-started/summaryofchanges.md [hunk 1]
```diff
@@ -52,26 +52,25 @@ The following features and enhancements were added.
...
+- The ngx-color picker has been replaced by a custom hue selection bar, lightness swatches bar, and color palette, allowing for a more customizable personalization experience [#235](https://github.com/zowe/zlux-app-manager/pull/235)
+- In this version, cross-launch via URL has been implemented, allowing for integration between Zlux and other web applications. This feature enables users to bookmark a set of app2app communication actions (in the form of a URL) that will be executed when opening the webpage [#234](https://github.com/zowe/zlux-app-manager/pull/234)
+- Improvements have been made to the bookmarking functionality in the TN3270 emulator [#30](https://github.com/zowe/tn3270-ng2/pull/30)
+  - Codepages have been reorganized so that the numbers are shown first, making it easier for users to navigate to their favorites
+  - Users can now save connection preferences on a per-user level. Clicking the floppy disk icon saves user settings to that user's scope.
+  - The buttons found in this feature have been realigned  
+- Several features have been added to the Zlux Editor [#153](https://github.com/zowe/zlux-editor/pull/153)
```
Comment (DivergentEuropeans) [inline docs/getting-started/summaryofchanges.md:62]: Sorry, our repo name zlux-editor is misleading. I believe on the Docs site, we refer to it simple as "Editor" or "Zowe Editor" https://docs.zowe.org/stable/user-guide/mvd-using.html#editor

File: docs/getting-started/summaryofchanges.md [hunk 2]
```diff
@@ -52,26 +52,25 @@ The following features and enhancements were added.
...
-  - Converted login activity to event emitter
-  - Added new local storage listener so changes can be reported across tabs
-  - Added detect activity when loging in from a new window/tab
-  - In idleWarn popup added
-  - Being active on the Desktop, while the session expiration prompt is active, will now dismiss the pop-up and attempt to renew the session. This feature also stays synchronized between multiple tabs, and will renew multiple tabs if the user is active in at least one of them
-- NGX-monaco has been removed [#155](https://github.com/zowe/zlux-editor/pull/155)
+- Login activity and session activity is now synchronized across multiple desktop tabs [#242](https://github.com/zowe/zlux-app-manager/pull/242)
+  - When a user logs out of a desktop tab, all other active tabs will also log out
+  - When a user performs an action on a desktop tab, the other tabs register this activity, which stops them from timing out
+- NGX-monaco has been removed due to a bug that would not allow the Editor to be opened more than twice [#155](https://github.com/zowe/zlux-editor/pull/155)
```
Comment (DivergentEuropeans) [inline docs/getting-started/summaryofchanges.md:70]: This should be titled "ngx-monaco-editor library" 
and changed to
"that would now allow the Editor to open files on the third instance" (Editor wasn't broken, but the file viewing component was)

File: docs/getting-started/summaryofchanges.md [hunk 3]
```diff
@@ -144,6 +143,7 @@ The following bugs were fixed.
 - Bugfix: An Infinite Auth loop would occur on explorer apps due to APIML and z/OSMF auth timeouts missmatch. In this version, the issue has been resolved by adding a force login flag if a datasets request comes back as 401 [#124](https://github.com/zowe/explorer-mvs/pull/124)
 - Bugfix: When using the JES Explorer to view Spool files of a job, users cannot open a spool file that has the same name as one already open. This issue has been resolved by adding a unique id to content tabs to allow opening of overlapping names [#188](https://github.com/zowe/explorer-jes/pull/188)
 - Bugfix: The `Env var` for `TERM` gets set to "linux", which is not recognized by USS. This issue has been resolved through the removal of rxjs-compat [#29](https://github.com/zowe/vt-ng2/pull/29)
+- Bugfix: NGX-monaco has been removed due to a bug that would not allow the Editor to be opened more than twice [#155](https://github.com/zowe/zlux-editor/pull/155)
```
Comment (DivergentEuropeans) [inline docs/getting-started/summaryofchanges.md:146]: ngx-monaco-editor library

Comment (DivergentEuropeans) [reply docs/getting-started/summaryofchanges.md:146]: and same as above

File: docs/getting-started/summaryofchanges.md [hunk 4]
```diff
@@ -52,26 +52,25 @@ The following features and enhancements were added.
...
-- Cross-launch via URL has been implemented [#234](https://github.com/zowe/zlux-app-manager/pull/234)
-- Improvements to feature and bookmarking functionality [#30](https://github.com/zowe/tn3270-ng2/pull/30)
-- Added close all + undo settings + hotkeys [#153](https://github.com/zowe/zlux-editor/pull/153)
+- The app server is now able to use more than one certificate authority (CA). This allows the server to validate other server's authenticity by recognizing the CA that another server may have used [#128](https://github.com/zowe/zlux-app-server/pull/128)
+- The ngx-color picker has been replaced by a custom hue selection bar, lightness swatches bar, and color palette, allowing for a more customizable personalization experience [#235](https://github.com/zowe/zlux-app-manager/pull/235)
+- In this version, cross-launch via URL has been implemented, allowing for integration between Zlux and other web applications. This feature enables users to bookmark a set of app2app communication actions (in the form of a URL) that will be executed when opening the webpage [#234](https://github.com/zowe/zlux-app-manager/pull/234)
```
[the remark this answers is not in the document]
Comment (DivergentEuropeans) [reply docs/getting-started/summaryofchanges.md:57]: or "apps" depending on whether or not we capitalize our term in other areas of the doc

File: docs/getting-started/summaryofchanges.md [hunk 5]
```diff
@@ -52,26 +52,25 @@ The following features and enhancements were added.
...
-  - Converted login activity to event emitter
-  - Added new local storage listener so changes can be reported across tabs
-  - Added detect activity when loging in from a new window/tab
-  - In idleWarn popup added
-  - Being active on the Desktop, while the session expiration prompt is active, will now dismiss the pop-up and attempt to renew the session. This feature also stays synchronized between multiple tabs, and will renew multiple tabs if the user is active in at least one of them
-- NGX-monaco has been removed [#155](https://github.com/zowe/zlux-editor/pull/155)
+- Login activity and session activity is now synchronized across multiple desktop tabs [#242](https://github.com/zowe/zlux-app-manager/pull/242)
+  - When a user logs out of a desktop tab, all other active tabs will also log out
+  - When a user performs an action on a desktop tab, the other tabs register this activity, which stops them from timing out
+- NGX-monaco-editor library has been removed, allowing the Editor to open and view files after the second instance of opening them [#155](https://github.com/zowe/zlux-editor/pull/155)
```
[the remark this answers is not in the document]
Comment (1000TurquoisePogs) [reply docs/getting-started/summaryofchanges.md:70]: Actually, this is duplicated in the bugfix section below.

File: docs/getting-started/summaryofchanges.md [hunk 6]
```diff
@@ -52,26 +52,25 @@ The following features and enhancements were added.
...
   - Removed use of node-sass, so that native compilation is not required
-  - Updated to typescript 3.7
-  - Updated to monaco 0.20
+  - Updated to typescript 3.7 from version 2.7.2
+  - Updated to monaco 0.2 from version 0.13. The monaco changelog can be found [here](https://github.com/microsoft/monaco-editor/blob/master/CHANGELOG.md)
```
Comment (1000TurquoisePogs) [inline docs/getting-started/summaryofchanges.md:73]: I believe 0.2 exists and is not the same as 0.20, please keep it 0.20 for correctness.

**Pull request merged**